app/components/action-feed/component.js

```diff
@@ -12,15 +12,57 @@ import { translationMacro as t } from 'ember-i18n';
  * ```
  * @class action-feed
  */
+
 export default Component.extend({
     store: service(),
     toast: service(),
-
+    currentUser: service(),
     classNames: ['action-feed'],
+    page: 1,
+    listLoading: false,
 
     errorMessage: t('components.action-feed.error_loading'),
 
     moreActions: computed('totalPages', 'page', function() {
         return this.get('page') < this.get('totalPages');
     }),
+
+    init() {
+        this._super(...arguments);
+        this.loadPage();
+        this.set('listLoading', true);
+        this.actionsList = [];
+    },
+
+    loadPage() {
+        const page = this.get('page');
+        this.get('currentUser.user').then(user => this.get('store').queryHasMany(user, 'actions', { page }).then(
+            response => this._setPageProperties(response)
+            , () => {
+                // Error
+                this._handleLoadError();
+            },
+        ));
+    },
+
+    _handleLoadError() {
+        this.set('listLoading', false);
+        this.set('loadingMore', false);
+        this.get('toast').error(this.get('errorMessage'));
+    },
+
+    _setPageProperties(response) {
+        this.setProperties({
+            actionsList: this.get('actionsList').concat(response.toArray()),
```

Comment:
I think it would make more sense to append to `actionsList` using `pushObjects`, rather than creating and assigning a new array.